node/blockchain/models/block/manager.py

```diff
@@ -38,7 +40,8 @@ def add_block_from_block_message(
         blockchain_facade: BlockchainFacade,
         *,
         signing_key: Optional[SigningKey] = None,
-        validate=True
+        validate=True,
+        expect_locked=True
```

Comment:
do not add this keyword argument, instead pop() it from kwargs somewhere in `TableLocker`

Reply:
Done.